ui/tests/integration/components/tools/wrap-test.js

```diff
@@ -70,7 +70,7 @@ module('Integration | Component | tools/wrap', function (hooks) {
     this.server.post('sys/wrapping/wrap', (schema, { requestBody, requestHeaders }) => {
       const payload = JSON.parse(requestBody);
       assert.propEqual(payload, JSON.parse(this.wrapData), `payload contains data: ${requestBody}`);
-      assert.strictEqual(requestHeaders['X-Vault-Wrap-TTL'], '30m', 'request header has default wrap ttl');
+      assert.strictEqual(requestHeaders['x-vault-wrap-ttl'], '30m', 'request header has default wrap ttl');
```

Comment:
I double checked and the header keys have the proper casing when passed to the client but somewhere internally they must convert to lower case.

Reply:
This has been an issue before in the tests.

Reply:
It should always be lowercase now with this new client at least. I'd like to move away from using mirage in this manner and use sinon instead and then we can assert that the headers are passed to the given api method as expected.